src/app/ApplicationTemplate.Shared.Views/App.xaml.cs

```diff
@@ -80,66 +77,50 @@ private static string GetContentRootPath()
 
 	private static string GetSettingsFolderPath()
 	{
+		var folderPath = string.Empty;
+
 //-:cnd:noEmit
 #if WINDOWS
-//+:cnd:noEmit
-		var folderPath = Windows.Storage.ApplicationData.Current.LocalFolder.Path; // TODO: Tests can use that?
-//-:cnd:noEmit
-#elif __ANDROID__ || __IOS__
-//+:cnd:noEmit
-		var folderPath = Environment.GetFolderPath(Environment.SpecialFolder.MyDocuments);
-//-:cnd:noEmit
+		folderPath = Windows.Storage.ApplicationData.Current.LocalFolder.Path; // TODO: Tests can use that?
```

Comment:
Is this still good for WinUI apps? (It might have been specific to UWP apps)

Reply:
No compilation error on my side with Windows, must be good but I'll test it, may be Microsoft bring this with WinUI or Uno made it compatible?

Reply:
It wouldn't be Uno because it's not used on Windows.

Reply:
It works!

``` text
"C:\\Users\\thomas.labrecque\\AppData\\Local\\Packages\\752f6418-d9b7-4185-ae8c-79d8ddbcabcd_s7k0xtv816k5a\\LocalState"
```